react/features/welcome/components/AbstractWelcomePage.js

```diff
@@ -101,6 +101,7 @@ export class AbstractWelcomePage extends Component<Props, *> {
      */
     componentDidMount() {
         this._mounted = true;
+        sendAnalytics(createWelcomePageEvent('viewed', undefined, { value: 1 }));
```

Comment:
is this desired on mobile too?

Reply:
Isn't it? My thinking was that if I'm putting it on web, why not on mobile too?

Reply:
I don't know, that's why I'm asking :-)